device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/Simulator.kt

```diff
@@ -615,6 +615,12 @@ class Simulator (
         return result.isSuccess
     }
 
+    override fun openUrl(url: String) : Boolean {
+        val command = listOf("xcrun", "simctl", "openurl", udid, url)
```

Comment:
what will happen if url contains '&' symbol?

Reply:
fixed by adding the quotes around